src/slide-potentiometer-element.stories.ts

```diff
@@ -0,0 +1,20 @@
+import { html } from 'lit-html';
+import { action } from '@storybook/addon-actions';
```

Comment:
unused import? or did you want to attach an action to the potentiometer input event?

Reply:
I never became good friends with storybook, what would be the advantage of adding the action on input event?  :)

Reply:
It means that when you move the potentiometer, you'll see the events firing in the "Actions" tab:

https://user-images.githubusercontent.com/892318/119320037-cf531b80-bc83-11eb-8d2a-1f1bff55bff2.mp4

Reply:
I see. Still not 100 % what you get from that, but it is quick to wire up, so I have done it.

Reply:
Thank you! 

It's just a quick way to test that the component emits the events correctly. There's an issue where it doesn't show the `detail` value of the events, otherwise this would also show users the information contained in events.